examples/swift/hello_world/ViewController.swift

```diff
@@ -15,7 +38,7 @@ final class ViewController: UITableViewController {
     super.viewDidLoad()
     do {
       NSLog("starting Envoy...")
-      self.client = try StreamClientBuilder().build()
+      self.client = try StreamClientBuilder().addFilter(PlatformDemoFilter()).build()
```

Comment:
You're right, the current implementation uses a single long-lived filter instance, which is a departure from upstream Envoy. This approach is definitely simpler to map, but I agree that aligning with upstream Envoy would be nice. Let me think a little bit about what it would take to make filter instances per-stream.

Reply:
👍 I think this is actually a hard requirement for filters in Envoy Mobile. I know a lot of the use cases Lyft has for platform filters today require filters to be per-stream and to act as both request and response filters

Reply:
Per discussion Friday, we'll do this on top of this PR.